doc/md/guides/database-per-tenant/02-db-groups.mdx

```diff
@@ -0,0 +1,323 @@
+---
+title: Defining Target Groups
+id: target-groups
+slug: target-groups
+---
+In Atlas, a target group is a collection of target databases whose schema is managed together. In a database-per-tenant
+architecture, each tenant's database is a target database, and all tenant databases are grouped into a target group.
+However, you can also group databases by other criteria, such as environment (dev, staging, prod), region, or any other
+criteria that makes sense for your application.
```

Comment:
```suggestion
conditions that makes sense for your application.
```

Reply:
group by condition sounds weird

Reply:
So anything else cause you are using 'criteria' in the beginning of the sentence